themes/default/content/registry/packages/azure-native/version-guide.md

```diff
@@ -73,6 +73,25 @@ keyvault.NewVault(/*...*/)
 
 {{% /choosable %}}
 
+{{% choosable language java %}}
+
+Currently no support for versions in Java.
```

Comment:
```suggestion
Java is not supported.
```

we try and avoid the use of the word _currently_ because it implies it will happen in the future, which isn't always true